client/layout/guided-tours/tours/media-basics-tour.js

```diff
@@ -122,24 +127,21 @@ export const MediaBasicsTour = makeTour(
 		<Step name="done" placement="center">
 			<p>
 				{ translate(
-					'Need to adjust your image? Click {{icon/}} {{strong}}Edit Image{{/strong}} to perform basic tweaks.',
+					'Need to adjust your image? Click {{icon/}} {{editImageButton/}} to perform basic tweaks.',
 					{
 						components: {
 							icon: <Gridicon icon="pencil" />,
-							strong: <strong />,
+							editImageButton: <strong>{ EDIT_IMAGE_BUTTON_LABEL }</strong>,
 						},
 					}
 				) }
 			</p>
 			<p>
-				{ translate(
-					'Click {{strong}}Done{{/strong}} to go back to your full library. Happy uploading!',
-					{
-						components: {
-							strong: <strong />,
-						},
-					}
-				) }
+				{ translate( 'Click {{doneButton /}} to go back to your full library. Happy uploading!', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Click {{DoneButton /}} to go back to your full library. Happy uploading!' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).